test/helpers/payment_details_helper_test.rb

```diff
@@ -92,14 +92,15 @@ class PaymentDetailsHelperTest < DeveloperPortal::ActionView::TestCase
                                          billing_address_country: 'country')
     stubs(:current_account).returns(account)
     stubs(:logged_in?).returns(true).once
+    stubs(:country_code_for).with('country').returns('ES').once
```

Comment:
Why do we need this `once` here?

Reply:
I like to use it in general, to prevent unexpected extra calls